modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,64 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The following table outlines supported features of primary and secondary networks created by `UserDefinedNetwork` (UDN) and other secondary networks. For more information on, the limitations of user-defined networks see: xref:../networking/ . For more information on secondary networks that are created through network attachement defintions using multus, see xref:../networking/.
+
+// column 4 and 8 will need put back in once supported
+[cols="<.^3,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1"]
+|====
+|Network type
+|definition
+|role
+//| services
+|east-west traffic
+|north-south traffic
+|Persistent IPs
+//|`NetworkPolicy`
+|`MultiNetworkPolicy`
+|Ingress
```

Comment:
By ingress here, do we mean OpenShift [ingress](https://docs.openshift.com/container-platform/4.16/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-using-ingress-and-routes_configuring-ingress-cluster-traffic-ingress-controller) ? If so, I have mislead you. We don't support that on secondary UDNs. 

I also don't understand the separation between a secondary UDN and a secondary network. These should be the same.

Reply:
@maiqueb , the matrix doesn't have ingress X for secondary networks created using UDN. The reason that we are separating out secondary UDN and secondary network is to demonstrate the difference between features for creating network segmentation using the UDN resource vs a NAD resource. 

These tables are tough to keep track of but I put a preview in the top comment: https://81484--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/understanding-multiple-networks.html#support-matrix-for-primary-secondary-networks_understanding-multiple-networks

Reply:
I get that; what I'm trying to say is I think the feature set on the secondary UDN is the same, independently of how it was created.

Reply:
copy, so I should add the same set of (x)s there too. Will do.

Reply:
@maiqueb updated. How's that look?

Reply:
I think you're missing east / west traffic for secondary networks - I would define it as `X [1][3]` .

Also, `Ingress` (assuming its OpenShift Ingress / routes) is **not** supported for secondary networks. In *any* topology.

Finally, I find it confusing to mark multi-network policy as available on all topologies; are we indicating we now support L3 secondary networks ? IIUC, the documentation was added in 4.13 / 4.14 but never shown to the user. Hence, I would also define it as `X [1][3]` .